config/default.json

```diff
@@ -5,8 +5,11 @@
     "settings": {
         "adminAccounts": "fakepassword"
     },
+    "api_key" : {
+        "SENDGRID": "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"
```

Comment:
the key shouldn't be in the codebase.

Reply:
Normally, when we use a third-party service, they give two keys. One for production and one for development (called "sand-box"). The sandbox key can be in the codebase, but not the production one. 
When there is only one key, it means it's for production and it shouldn't be in the codebase. 

So to test development, we just use `export` on the terminal for testing for the moment.